libs/platform/src/lib/utils/object.ts

```diff
@@ -1,3 +1,13 @@
 export function getNestedValue(key: string, object: any): any {
-    return key.split('.').reduce((a, b) => a[b], object);
+    return key
+        .split('.')
+        .reduce((a, b, i, arr) => {
+            // Break loop if we cannot dive inside
+            if (!a[b]) {
+                // Mutating original array will break reduce
+                arr.pop();
+            }
```

Comment:
For what is it?
If there is no `b` in `a` we can return `undefined` without array modification.

Reply:
`getNestedValue('x.y.z', {})` will throw `Uncaught TypeError: Cannot read property ...` anyway.

Reply:
You right, we cannot break reduce so regular loop used here.